assets/js/country_support.js

```diff
@@ -3,7 +3,9 @@
  * @property {string} name
  * @property {string} country_code The international dialing code (ex "1" for the US)
  * @property {boolean} supports_sms
+ * @property {boolean?} supports_sms_unconfirmed
  * @property {boolean} supports_voice
+ * @property {boolean?} supports_voice_unconfirmed
```

Comment:
Just to confirm, this is expecting the value to be either `boolean` or `null`. If (based on the specs) it's instead either `boolean` or `undefined` / absent, then I'd expect:

```suggestion
 * @property {boolean=} supports_sms_unconfirmed
 * @property {boolean} supports_voice
 * @property {boolean=} supports_voice_unconfirmed
```

Reply:
Yup, that was exactly what I meant. Thanks!